pages/index-new.tsx

```diff
@@ -0,0 +1,112 @@
+import LearnSection from "components/front-page/LearnSection";
```

Comment:
What's this page for?

Reply:
I created this temporarily so that the existing home page can remain while I continue adding components to the new home page (index-new) and be able work on the layout. When it's approved and good to go I'll remove the existing index page and rename _index-new_ to _index._

Reply:
Only issue I see with this approach is that if anything changes on the index page (I have some changes here: https://github.com/zeitgeistpm/ui/pull/544) we'll need to make sure they are pulled into `index-new` and we won't really have git to help us. Might be easier to build of the components first then compose them once they're ready. Happy for you to make the call though

Reply:
I see what you're saying. I think I'll keep it though so I can do some testing on the layout itself for now but after I'll update the original index page instead once we are ready to make the switch